curp-test-utils/src/test_cmd.rs

```diff
@@ -154,6 +154,37 @@ impl TestCommand {
     }
 }
 
+/// LogIndex used in Command::ASR
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq)]
+pub struct LogIndexResult(LogIndex);
```

Comment:
Because `LogIndex` is actually a primitive `u64`. The `PbSerialize` is only intended for test use, I need to wrap it in a new type and impl `PbSerialize` for it.